### ISSUE TITLE
logging output with sequence name

### ISSUE THREAD

**Issue by dlaehnemann** [hzi-bifo/mdasim#11]
The current logging output was implemented with only one starting sequence in the input `fasta` file in mind, as this was our use case at the moment. However, `mdasim` also accepts input `fasta` files with mutliple sequence inputs. For introduced errors to be logged usefully, we then obviously need to also track which sequence of origin an error refers to and include a new column in the logging output file.

**Comment by VSack** [comment]
Here's a thing: if I run MDAsim with an input fasta file with 2 sequences like
```
>chromosome:A
GATCATGCAGCTCTTCCAGGCCCACTGCTTCTTCCTGTCCACTAGGCCACAGCCGCCCTC
...
>chromosome:B
GATCATGCAGCTCTTCCAGGCCCACTGCTTCTTCCTGTCCACTAGGCCACAGCCGCCCTC
...
```
And I let it print the names of the sequences, I get
```
**************************************
Loading reference DNA sequence ...
chromosome:A
DNA sequence loaded.
```
Where actually, it should find `chromosome:B` as well. If I run a test with one sequence being only Gs and one being only As, it produces G- and C- sequences only. I ran this with 2.0, but without the logging, which means, it should actually do just the same as 1.2, since we didn't change the sequence loading and so on.
**Thus**: Are we sure that MDAsim 1.2 could handle fasta files with more than one sequence in them at all? I mean, the paper states
> The inputs of the simulator are DNA strand(s), the list of primers, both in fasta format, ...

But are we sure it could do this?

**Comment by dlaehnemann** [comment]
Well, no...

And I guess for our own use of mdasim a single input sequence is fine. We then just need to document this properly. And eventually, it would be good if mdasim complained when provided a fasta file with multiple entries, i.e. that it notified users that it will only use the first of those sequences. Silently doing that is just asking for trouble.

So the TODO here would be:

* [x] update the docs accordingly
* [x] see if and where we can implement a quick check during the sequence loading, e.g. if anything else comes in the provided file after the (first) sequence has been read

**Comment by dlaehnemann** [comment]
I am assuming, that there is no quick and easy way to check for further sequences in the input fasta file beyond the first sequence? Or should we reopen this issue for the second TODO above?

**Comment by VSack** [comment]
Oh I did fix that in 6f5cbee119da4aa6c18b27d6f8bd7f1c08b1773b, sorry, should have referenced this issue. The software now exits with an error message when there is more than one sequence in the input file. Actually using more than one sequence in the simulation would require a lot more changes throughout the code though, so for now I think this is the best solution, since it's the same as v1.2 did but more clear to the user.

**Comment by dlaehnemann** [comment]
Ah, perfect. Thanks for referencing it now. And yes, I think this is the best solution for now--and generally I also like the idea of a tool that does one atomic job and does it well. If you want more sequences amplified, it also doesn't hurt to parallelize it through separate runs of mdasim, so it's all good.

And then I guess this leaves issue #12 to be resolved before we can bundle up a new minor bugfix release (`2.0.2`).
